### ISSUE TITLE
Blazor Webassembly .dll files cannot be downloaded

### ISSUE THREAD

**Issue by calnastic0** [dotnet/aspnetcore#44784]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When the solution is running in debugging mode, it is good. But was making a deployment to my dev computer's IIS. And by accessing "localhost/app", the antivirus software blocked the download of all .dll files. 

Searched around the internet, one solution from [here](https://github.com/dotnet/aspnetcore/issues/5477) is to rename all .dll files to .bin, plus modify/delete related files. However the antivirus blocked all .bin files this time. Also the same topic was discussed [here](https://github.com/dotnet/aspnetcore/issues/21996) regarding Sophos blocking .dll files. It is exactly what my company is using. 

Also tried to do an AOT compilation, only to find the deployment package still consists of .dll files. 

I know this is not a new topic but no available solutions seem to work. It has been a long time since the problem emerges. I'm hoping some work has been done toward it.

Thank you.

### Expected Behavior

Would expect to be able to run the webassembly client in the browser without being blocked by antivirus.

### Steps To Reproduce

To reproduce, just use one of the antivirus software, such as Sophos, and deploy an out-of-box solution created by .net 6 Blazor webassembly.

![image](https://user-images.githubusercontent.com/58486299/198702594-3e9b90c7-0553-4952-a993-44174ccaa05d.png)

### Exceptions (if any)

All .dll extension files were unable to download to browser.

### .NET Version

6.0.402

### Anything else?

Microsoft Visual Studio Community 2022
Version 17.3.6
VisualStudio.17.Release/17.3.6+32929.385
Microsoft .NET Framework
Version 4.8.04084

.NET SDK (reflecting any global.json):
 Version:   6.0.402
 Commit:    6862418796

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.402\

**Comment by TanayParikh** [comment]
Hey @calnastic0, thanks for reaching out.

Have you had a chance to check out: https://devblogs.microsoft.com/dotnet/custom-deployment-layout-for-blazor-webassembly-apps/? It contains some tips which may be helpful in your scenario.

**Comment by calnastic0** [comment]
> Hey @calnastic0, thanks for reaching out.
> 
> Have you had a chance to check out: https://devblogs.microsoft.com/dotnet/custom-deployment-layout-for-blazor-webassembly-apps/? It contains some tips which may be helpful in your scenario.

Hi TanayParikh,

Thanks for the reply. I actually have come across this solution, but after checking on the [nuget ](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.MultipartBundle) page it says "Using this package in production is not supported." Do you know what this alert means? If it is used for production, what could be the potential problems?

**Comment by TanayParikh** [comment]
It's an experimental, pre-release package, which is not yet officially supported. More details, and tracking the productization of that package is covered in https://github.com/dotnet/aspnetcore/issues/36978.

**Comment by calnastic0** [comment]
> It's an experimental, pre-release package, which is not yet officially supported. More details, and tracking the productization of that package is covered in #36978.

Thank you, also in that post another solution BlazorWasmAntivirusProtection was introduced, and it gets more frequently updated in comparison to MultipartBundle. Let us evaluate these two approaches.

**Comment by TanayParikh** [comment]
Marking this as a duplicate of https://github.com/dotnet/aspnetcore/issues/36978.
